src/components/modals/ManageAntModal/ManageAntModal.tsx

```diff
@@ -0,0 +1,236 @@
+      onClick={handleClickOutside}
+    >
+      <div className="flex-column" style={{ margin: '10%' }}>
+        <div className="flex flex-row">
+          <span
+            className="flex section-header"
+            style={{
+              justifyContent: 'flex-start',
+              width: '100%',
+            }}
+          >
+            <NotebookIcon width={25} height={25} fill={'var(--text-white)'} />
+            &nbsp;Manage ANT:&nbsp;
+            <span>
+              {isMobile ? (
+                <CopyTextButton
+                  displayText={`${contractId.slice(0, 10)}...${contractId.slice(
+                    -10,
+                  )}`}
+                  copyText={contractId}
+                  size={24}
+                  wrapperStyle={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    textColor: 'var(--bright-white)',
+                  }}
+                />
+              ) : (
+                <CopyTextButton
+                  displayText={contractId}
+                  copyText={contractId}
+                  size={24}
+                  wrapperStyle={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    textColor: 'var(--bright-white)',
+                  }}
+                />
+              )}
+            </span>
+          </span>
+        </div>
+        <table className="assets-table">
+          <RowItem
+            details={[
+              'Status:',
+              <TransactionStatus
+                key={`${contractId}-confirmations`}
+                confirmations={confirmations}
+              />,
+            ]}
+            bgColor={'#1E1E1E'}
+            textColor={'var(--text-white)'}
+          />
+          <RowItem
+            details={[
+              `Associated Names: ${associatedNames.length}`,
+              associatedNames
+                ? associatedNames.map((name) => (
+                    <>
+                      <span
+                        className="assets-manage-button"
+                        key={`${contractId}-name`}
+                      >
+                        {name}
+                      </span>
+                      &nbsp;
+                    </>
+                  ))
+                : 'N/A',
+            ]}
+            bgColor={'#1E1E1E'}
+            textColor={'var(--text-white)'}
+          />
+          <RowItem
+            details={[
+              'Nickname:',
+              state?.name ? state.name : 'N/A',
+              <button
+                className="button"
+                key={`${contractId}-nickname-edit-button`}
+              >
+                <PencilIcon width={20} height={20} fill="var(--text-white)" />
+              </button>,
+            ]}
+            bgColor={'#1E1E1E'}
+            textColor={'var(--text-white)'}
+          />
```

Comment:
nit: can we crate an object and use it to dynamically create these `RowItems`


e.g.
```
const antDetails = {
    'Nickname':  state?.name ? state.name : 'N/A',
    'Ticker': state?.ticker ? state.ticker : 'N/A',
    'Target ID': targetId ? targetId : 'N/A',
    ....etc.
}
```

and then use it like
```
Object.entries(antDetails).map(([key, value], index) => (
    <RowItem
            details={[
              key,
              value
              <button
                className="button"
                key={`${contractId}-${key}-edit-button`}
              >
                <PencilIcon width={20} height={20} fill="var(--text-white)" />
              </button>,
            ]}
            bgColor={'#1E1E1E'}
            textColor={'var(--text-white)'}
          />
)
```

Reply:
for reference - we are doing something similar to ☝️  in the `AntCard` as it displays similar details
https://github.com/ar-io/arns-react/blob/develop/src/components/cards/AntCard/AntCard.tsx#L99-L108

Reply:
updated